Tests/iaas/image-metadata/image-md-check.py

```diff
@@ -217,11 +329,45 @@ def report_stdimage_coverage(imgs):
     return err
 
 
+def miss_replacement_images(images, outd_list):
+    """Go over list of images to find replacement imgs for outd_list, return the ones that are left missing"""
+    rem_list = []
+    for outd in outd_list:
+        success = False
+        last_spc = outd.rfind(" ")
+        shortnm = outd
+        if last_spc != -1:
+            shortnm = outd[:last_spc]
+        for imgnm in images:
+            # Skip over other images
+            if imgnm != outd and imgnm != shortnm:
+                continue
+            # Skip over itself
+            if imgnm == outd:  # or success:
+                continue
```

Comment:
I guess we can move this up, and then the check above becomes `if imgnm != shortnm:`
Also, if no space is found in the image name, then we can skip this whole for loop altogether.